Add Files Lifecycle occ command set (#3607)
owncloud/docs: modules/admin_manual/pages/enterprise/file_management/files_lifecycle.adoc

To stay informed, users can also receive regular Activity summaries by e-mail. For auditing purposes, the extension integrates with the {oc-marketplace-url}/apps/admin_audit[Auditing] app to provide all events of interest in the logs.

== Setup & Configuration

See the xref:configuration/server/occ_command.adoc#file-lifecycle-management[lifecycle occ command set] for details when using the command line.

=== Archive Location

By default, archived files are stored within the ownCloud data directory, but outside the users files directories so that they are not accessible using the Web UI and other clients.

[width="70%",cols="40%,80%",options="header"]
|===
| Type 
| Location
| User files
| `/<datadir>/$userid/files`
| Archived files
| `/<datadir>/$userid/archive/files/`
|===

=== Setting Upload Times for Existing Files

File Lifecycle Management uses the _upload time_ of files (server time at which it first appeared on the ownCloud server) to determine when to archive them. As ownCloud Server generally does not store this metadata, the File Lifecycle Management extension takes care of this when it is enabled.

> When File Lifecycle Management is set up on an existing ownCloud installation, you therefore have to set an _upload time_ for all files that existed before the extension has been enabled or when it was temporarily disabled to allow the archiving policies to do their job. To set an upload time for all files that do not yet have one, you can use the _occ_ command `lifecycle:set-upload-time`. 

Reviewer: ```suggestion
When File Lifecycle Management is set up on an existing ownCloud installation, you therefore have to set an _upload time_ for all files that existed before the extension has been enabled. The same applies if it was temporarily disabled. Only then can the archiving policies work. To set an upload time for all files that do not yet have one, you can use the _occ_ command `lifecycle:set-upload-time`. 
```

TIP: Files without an _upload time_ will not be considered for archiving.

> TIP: You only have to conduct this process once when setting up File Lifecycle Management on an installation with existing files or when it was temporarily disabled . Files added after enabling File Lifecycle Management will be tracked automatically.

Reviewer: ```suggestion
TIP: You only have to conduct this process once when setting up File Lifecycle Management on an installation with existing files or if it was temporarily disabled . Files added after enabling File Lifecycle Management will be tracked automatically.
```
